libs/java/auth_core/src/main/java/com/yahoo/athenz/auth/Authority.java

```diff
@@ -197,4 +200,8 @@ default String getUserEmail(final String username) {
     default List<Principal> getPrincipals(EnumSet<Principal.State> principalStates) {
         return Collections.emptyList();
     }
+
+    default String getCAPathProperty() {
```

Comment:
I'm not sure why we have this method in the interface? I expect this type of functionality be within the authority implementation itself and not in the interface.